junction/base/constants.py

```diff
@@ -51,3 +51,14 @@
 PROPOSAL_USER_VOTE_ROLES = ((1, PROPOSAL_USER_VOTE_ROLE_PUBLIC),
                             (2, PROPOSAL_USER_VOTE_ROLE_REVIEWER),
                             )
+
+PROPOSAL_REVIEW_VOTE_MUST_HAVE = "Must have"
+PROPOSAL_REVIEW_VOTE_GOOD = "Good"
+PROPOSAL_REVIEW_VOTE_NOT_BAD = "Not Bad"
+PROPOSAL_REVIEW_VOTE_NOT_ALLOWED = "Shouldn't be allowed"
+
+PROPOSAL_REVIEW_VOTES_LIST = ((2, PROPOSAL_REVIEW_VOTE_MUST_HAVE),
+                              (1, PROPOSAL_REVIEW_VOTE_GOOD),
+                              (0, PROPOSAL_REVIEW_VOTE_NOT_BAD),
+                              (-1, PROPOSAL_REVIEW_VOTE_NOT_ALLOWED),
+                              )
```

Comment:
Awesome! Can we have these values in Database ? Say tomorrow some other conference needs to have one more state "Copied Content with vote value -2" they can't do it. 

Moving to the DB with default values as above will be good.